### PR TITLE
Mr/domain request form text updates

### PR DESCRIPTION
**Pull request by michelle-rago**
# Domain request form text updates #

## 🗣 Description ##

Made text changes to questions, explanatory text, legal content, and added school district org type

## 💭 Motivation and context ##

Updates are based on changes we made to the form

### CODE REVIEW

**Review by SSPJ — APPROVED** [review]
LGTM 👍  Thanks for these updates, @michelle-rago !

I'm doing an abbreviated review as I did not see anything which would impact security, accessibility, or code structure.

The Basics
- [x] I read the code
- [x] I ran the code

If you'd like a 👍 from Logan as well, make sure she knows you are waiting on her, otherwise, this is good to go! 💯

**Comment by michelle-rago** [comment]
No need for a second review. Thanks!

**Pull request merged**
